src/util/http/websocket/QueryId.h

```diff
@@ -111,5 +124,31 @@ class QueryRegistry {
     } while (!result.has_value());
     return std::move(result.value());
   }
+
+  /// Member function that acquires a read lock and returns a vector
+  /// of all currently registered queries.
+  std::vector<QueryId> getActiveQueries() const {
+    return registry_->withReadLock([](const auto& map) {
+      // TODO<C++23> Use ranges to transform map keys into vector
```

Comment:
```suggestion
      // TODO<C++23> Use `ranges::to` to transform map keys into vector
```
Maybe I'll implement a simple drop-in replacement for the time being because at least the `toVector` case happens EVERYWHERE (especially also in the tests).